upup/models/cloudup/resources/addons/networking.romana/k8s-1.6.yaml.template

```diff
@@ -0,0 +1,312 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: romana-agent
+rules:
+- apiGroups:
+  - "*"
+  resources:
+  - pods
+  - nodes
+  verbs:
+  - get
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: romana-agent
+  namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: romana-agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: romana-agent
+subjects:
+- kind: ServiceAccount
+  name: romana-agent
+  namespace: kube-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: romana-etcd
+  namespace: kube-system
+spec:
+  clusterIP: {{ .Networking.Romana.EtcdServiceIP }}
+  ports:
+  - name: etcd
+    port: 12379
+    protocol: TCP
+    targetPort: 4001
+  selector:
+    k8s-app: etcd-server
+  sessionAffinity: None
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: romana
+  namespace: kube-system
+spec:
+  clusterIP: {{ .Networking.Romana.DaemonServiceIP }}
+  ports:
+  - name: daemon
+    port: 9600
+    protocol: TCP
+    targetPort: 9600
+  selector:
+    romana-app: daemon
+  sessionAffinity: None
+  type: ClusterIP
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: romana-daemon
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        romana-app: daemon
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      hostNetwork: true
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: romana-daemon
+        image: quay.io/romana/daemon:v2.0-preview.2
+        imagePullPolicy: Always
+        args:
+        - --cloud=aws
+        - --network-cidr-overrides=romana-network={{ .KubeControllerManager.ClusterCIDR }}
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: romana-listener
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        romana-app: listener
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      hostNetwork: true
+      serviceAccountName: romana-listener
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: romana-listener
+        image: quay.io/romana/listener:v2.0-preview.2
+        imagePullPolicy: Always
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: romana-agent
+  namespace: kube-system
+spec:
+  template:
+    metadata:
+      labels:
+        romana-app: agent
+    spec:
+      hostNetwork: true
+      securityContext:
+        seLinuxOptions:
+          type: spc_t
+      serviceAccountName: romana-agent
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: romana-agent
+        image: quay.io/romana/agent:v2.0-preview.2
```

Comment:
Is this released in GA? If not may want to add a note to the docs

Reply:
Added a note into the Romana section of networking doc.